rules/windows/builtin/code_integrity/win_codeintegrity_attempted_dll_load.yml

```diff
@@ -104,6 +104,14 @@ detection:
         - FileNameBuffer|contains: '\Program Files\SentinelOne\Sentinel Agent'
         # Example: Program Files\SentinelOne\Sentinel Agent 23.4.4.223\SentinelAgent.exe
         - ProcessNameBuffer|contains: '\Program Files\SentinelOne\Sentinel Agent'
+    filter_optional_national_instruments:
+        # Example: \device\harddiskvolume3\program files\national instruments\shared\mdns responder\nimdnsnsp.dll
+        FileNameBuffer|contains: '\National Instruments\Shared\mDNS Responder\'
+    filter_optional_kaspersky:
+        # Example: \Program Files (x86)\Kaspersky Lab\Kaspersky Endpoint Security for Windows\x64\antimalware_provider.dll
+        FileNameBuffer|contains|all:
+            - 'antimalware_provider.dll'
+            - '\Kaspersky Lab\'
```

Comment:
the filter did not match the example log.
"kaspersky" is in the `ProcessNameBuffer `
the dll "\mdnsNSP.dll" is in `FileNameBuffer`

Reply:
Thanks, I will check that and fix.